test/unit/payload-parser_test.js

```diff
@@ -241,6 +241,27 @@ describe('Thinking things payload parser', function() {
             thinkingParser.parse('#STACK07#3,B,4.70,1,2,4,5,0,-1$', checkSleep('-1', '', done));
         });
     });
+    describe('When a Generic module arrives: #STACK01#19,GM,attrName,32,600$,', function() {
+        it('should fill the Module ID', function(done) {
+            thinkingParser.parse('#STACK01#19,GM,attrName,32,600$,',
```

Comment:
Maybe it would be nice to move the string '#STACK01#19,GM,attrName,32,600$,' to a variable and reuse it in the 3 unit tests of this suite... ;) Just to ease its updating in the feature... :)